nautobot/docs/plugins/region-and-site-data-migration-guide.md

```diff
@@ -0,0 +1,285 @@
+# Region and Site Related Data Model Migration Guide
+
+`Region` and `Site` models are collapsed into `Location` in Nautobot V2.0. As a result, all the `Region` and `Site` related data models are being migrated to use `Location` as well. Below is a comprehensive guide for Nautobot App developers to migrate their `Region` and `Site` related data models to `Location`.
```

Comment:
I feel like the first and second sentences here are redundant/repetitive but I'm not sure what to suggest as a rewrite.